sdk/bare/common/usr/demo/task_cc.c

```diff
@@ -95,9 +96,11 @@ void task_cc_callback(void *arg)
     while (&cc_ctxs[cc_idx] != ctx) cc_idx++;
 
     // ---------------------
-    // Update position based on user specified speed
+    // Update position
     // ---------------------
-    ctx->theta_e += ctx->omega_e * Ts;
+//    if (ctx->set_theta == 0){     // Update position based on user specified speed
+//        ctx->theta_e += ctx->omega_e * Ts;
+//    }
```

Comment:
Why is this commented out?

Reply:
This is by mistake, I have uncommented this part in the commit Id `57b85253ab78a3e7f768448ae8b2060f5e0c2b66`